envoy: Rebase to get gRPC proxy responses.

jrajahalme opened the pull request:
Rebase to the upstream Envoy commit that merged the gRPC responses to gRPC requests.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

rlenglet commented on envoy/WORKSPACE:
```diff
@@ -38,8 +37,8 @@ load("@com_lyft_protoc_gen_validate//bazel:go_proto_library.bzl", "go_proto_repo
 go_register_toolchains()
-load("@io_bazel_rules_go//proto:def.bzl", "proto_register_toolchains")
-proto_register_toolchains()
+# load("@io_bazel_rules_go//proto:def.bzl", "proto_register_toolchains")
```
I don't understand this particular change.
If that's legit, please delete the lines instead of commenting them.

jrajahalme replied:
Right, I did not see these lines in upstream WORKSPACE files, so decided to comment them out to see if they are (still) required. Will remove, as envoy builds in this PR.

jrajahalme committed:
Gopkg: Update golang/protobuf

Newly generated Envoy APIs require a newer golang/protobuf, maybe
because we use latest protoc-gen-go?

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

jrajahalme committed:
envoy: Rebase to get gRPC proxy responses.

Rebase to the upstream Envoy commit that merged the gRPC responses to gRPC requests.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>
